CMakeLists.txt

```diff
@@ -0,0 +1,151 @@
+#
+# CMakeLists.txt
+# Copyright (c) 2019 Serein Pfeiffer <serein.pfeiffer@gmail.com>
+#
+# This file is part of the Box2D QML plugin.
+#
+# This software is provided 'as-is', without any express or implied warranty.
+# In no event will the authors be held liable for any damages arising from
+# the use of this software.
+#
+# Permission is granted to anyone to use this software for any purpose,
+# including commercial applications, and to alter it and redistribute it
+# freely, subject to the following restrictions:
+#
+# 1. The origin of this software must not be misrepresented; you must not
+#    claim that you wrote the original software. If you use this software in
+#    a product, an acknowledgment in the product documentation would be
+#    appreciated but is not required.
+# 2. Altered source versions must be plainly marked as such, and must not be
+#    misrepresented as being the original software.
+# 3. This notice may not be removed or altered from any source distribution.
+#
+cmake_minimum_required (VERSION 3.9 FATAL_ERROR)
+project (qmlbox2d)
```

Comment:
Before, the lib was named libBox2D.so.
I'm not sure if we should keep this name for compatibility or update it to something like qmlbox2d so it really tells it is the qml port of box2d

Reply:
I think naming the QML plugin qmlbox2d instead is a good idea. :-)